eidas-saml-parser/src/dist/config.yml

```diff
@@ -16,17 +16,12 @@ logging:
   appenders:
     - type: ${LOGGING_APPENDER:-logstash-console}
 
-connectorMetadataConfiguration:
-  url: ${CONNECTOR_NODE_METADATA_URL:-https://DefaultNotUsed}
-  expectedEntityId: ${CONNECTOR_NODE_ENTITY_ID:-DefaultNotUsed}
-  jerseyClientName: connector-metadata-client
-  maxRefreshDelay: ${METADATA_REFRESH_DELAY:-600000}
-  trustStore:
-    type: ${TRUSTSTORE_TYPES:-encoded}
-    store: ${CONNECTOR_NODE_METADATA_TRUSTSTORE:-DefaultNotUsed}
-    password: ${CONNECTOR_NODE_METADATA_TRUSTSTORE_PASSWORD:-marshmallow}
+metatronUri: ${METATRON_URL:-https://DefaultNotUsed}
```

Comment:
Maybe drop the default? It's always going to be required I think.

Reply:
I figured that meant that there is no default/don't use a default judging by other places it shows up.  Delete the other `DefaultNotUsed`s?

Reply:
hmm..interesting. Not sure in that if that's the case. Seems weird to supply a default if the intention is to stop you using one. Surely _not_ providing one would force you to supply a value.

Reply:
I tend to agree and I've deleted defaultNotUsed wherever I found it.